### DISCUSSION TITLE
为什么歌名会被打码

### DISCUSSION THREAD

**Discussion by benzBrake** [qier222/YesPlayMusic#1337] [Q&A] [answered]
# 尽量每个 issue 只提一个 bug 或新功能
- 歌名是 Sexy Love
- 用的是客户端
- Windows 10

![bP88Ug.jpg](https://s4.ax1x.com/2022/02/23/bP88Ug.jpg)

**Comment by pan93412** [comment, marked as answer]
<img width="1042" alt="截圖 2022-02-23 下午10 54 27" src="https://user-images.githubusercontent.com/28441561/155344197-00d3ead9-56bf-4f05-bf62-ee7265d8457c.png">

https://music.163.com/#/album?id=2384020

那是網易雲打碼的。
